### DISCUSSION TITLE
Help - Error "couldn't find any D:"

### DISCUSSION THREAD

**Discussion by ECM56** [TheLastGimbus/GooglePhotosTakeoutHelper#252] [I NEED HELP] [answered]
Hi!  Just found your tool after downloading a large number of photos from Google Photos.
Working on a Windows 10 system.  Ultimate destination Synology Photos on a Synology NAS.  Synology KB led me to your repository.
I downloaded gpth-windoza.exe and put it in the existing Synology folder in \Program Files (x86).
I have extracted all the files from the ZIP files into a single folder named Takeout - C:\Takeout.
I have created an empty folder named Google-Photos-Out - C:\Google-Photos-Out.
I have tried running gpth-windoza.exe both from Powershell and by right clicking and executing it from File Manager.
I get what appear to be all the correct prompts and I get pop-up windows to select the input and output folders and the options.
(I also tried it from the command line - PS C:\Program Files (x86)\Synology\TakeoutHelper> .\gpth-windoza.exe --input "C:\Takeout" --output "C:\Google-Photos-Out" --albums "nothing" --divide-to-dates)
In all cases, after I select the folders and options I receive the following error:
...oh :(
...
I couldn't find any D: reasons for this may be:
  - your Takeout doesn't have any "year folders" -
    visit https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper
    again and request new, correct Takeout
After fixing this, go ahead and try again :)

I'm not sure I understand this error and why it is looking for D:.

I extracted all the files and pulled all the .jpg and .jpg.json files into the one Takeout folder - did I go wrong there?  Was I supposed to leave the year folders (such as "Photos from 2021")?

The folder structure in the Google ZIP files was Takeout/Google Photos/[Various Folder Names]/jpeg and json files.

I would appreciate any advice - I have 70 zip files from Google and about 64,000 .jpg and .jpg.json files (I suppose I should try a bite-sized chunk before spending the whole day, huh?)

Thank you!
Eric

**Comment by ECM56** [comment, marked as answer]
Hello!
I think the problem was as I suspected (User Error).
The error message is 100% correct.  While all the ZIP files should be extracted, you want to leave the rest of the file structure intact.
I just did a test with 1 ZIP and it worked without error.
69 more to go . . .
I hope this information might help someone else in the future!
Eric

**Reply by ECM56** [reply]
Confirming the entire job ran . . . I didn't time it but pretty quick - older machine and it was done in less than 20 minutes.

**Reply by TheLastGimbus** [reply]
Hi! Umm, there is a nice visual guide in readme on how to handle folder structure from multiple zips 👀

You should not be forced to do this one by one 🤔🤔
